### PR TITLE
PHX-15831 adding docker-agent collector configuration via environment…

### PR DESCRIPTION
**Pull request by nhatfield**
… variables

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -2,48 +2,75 @@
...
+
+	if [ "${APIURL}" ]; then
+		sed -i -e "s%url =.*%url = ${APIURL}%g" /opt/netuitive-agent/conf/netuitive-agent.conf
+		echo "Configuring URL: $APIURL"
+	fi
+
+	if [ ! "${STATSD_HOST}" ]; then
+		STATSD_HOST="localhost"
+	fi
+	sed -i -e "s/sensu\.app\.netuitive\.com/${STATSD_HOST}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
```
Comment (TheConnMan) [inline entrypoint.sh]: This doesn't seem like it should be in the public repo.

**Review by TheConnMan — COMMENTED** [review]
Generally looks good. Thanks for adding docs. I'd like a review by someone other that me as well.

File: entrypoint.sh (hunk 1 above)
Comment (nhatfield) [reply entrypoint.sh]: Good catch!

**Comment by rclagett** [comment]
I noticed the EV you added APIURL does not appear in the README. Could we add it there?

It appears HTTPVAR, APIHOST, and LIP are missing as well for some reason. I'm fine if we rather add these in a separate PR given you didn't add them to the entrypoint script but I wouldn't be opposed to adding them with these changes either - up to you!

**Review by rclagett — COMMENTED** [review]
Overall, looks really good, thanks! Just a couple questions in the comments.

**Review by rclagett — APPROVED** [review]
Looks great!

**Pull request merged**